cli/compose/schema/bindata.go

```diff
@@ -303,4 +327,3 @@ func _filePath(dir, name string) string {
 	cannonicalName := strings.Replace(name, "\\", "/", -1)
 	return filepath.Join(append([]string{dir}, strings.Split(cannonicalName, "/")...)...)
 }
-
```

Comment:
I think this might be why `validate` is failing. Do you have precommit hooks that remove this line?